From e0176c73d9e4cb5ead10c075123bc54385ebd148 Mon Sep 17 00:00:00 2001
From: Xylar Asay-Davis <xylarstorm@gmail.com>
Date: Fri, 14 Feb 2020 14:06:19 -0700
Subject: [PATCH] Add call to find_library for Windows

This seems to be necessary, at least for conda-forge CI.
---
 jigsawpy/libsaw.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/jigsawpy/libsaw.py b/jigsawpy/libsaw.py
index 278ee7c..94b80ae 100644
--- a/jigsawpy/libsaw.py
+++ b/jigsawpy/libsaw.py
@@ -1,5 +1,6 @@
 
 import ctypes as ct
+import ctypes.util
 import numpy as np
 import inspect
 import platform
@@ -61,7 +62,7 @@ if (JLIBNAME == Path()):
 if (JLIBNAME == Path()):
 #---------------------------- search machine path for binary
     if   (platform.system() == WIN):
-        JLIBNAME = Path("jigsaw.dll")
+        JLIBNAME = Path(ctypes.util.find_library("jigsaw.dll"))
 
     elif (platform.system() == LNX):
         JLIBNAME = Path("libjigsaw.so")
-- 
2.23.0

